### PR TITLE
Fix item visbility canceling each other out

### PR DESCRIPTION
**Pull request by bryce-seifert**
Fix for https://github.com/bitfocus/companion-module-obs-studio/issues/17

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -241,6 +247,34 @@ instance.prototype.updateScenes = function() {
...
+		}
+		self.actions();
+		self.init_feedbacks();
+		data.sources.forEach(source => {
+			self.states[source.name] = false;
+		});
+	});
+	self.obs.send('GetCurrentScene').then(data => {
+		data.sources.forEach(source => {
+            self.obs.send('GetSceneItemProperties',  { item: source}).then(data => {
```
Comment (krocheck) [inline index.js:266]: mix of tabs and spaces on this line

File: index.js [hunk 2]
```diff
@@ -241,6 +247,34 @@ instance.prototype.updateScenes = function() {
...
+				if ( data['visible'] == true) {
+					self.states[data['name']] = true;
+				} else {
+					self.states[data['name']] = false;
+				}
+				self.checkFeedbacks('scene_item_active');
+			});
+        });
```
Comment (krocheck) [inline index.js:274]: ditto

**Comment by krocheck** [comment]
Tested?

**Comment by bryce-seifert** [comment]
@krocheck Cleaned up the spaces. Did some more testing and found some weird exceptions for when in Studio Mode, added those above.

File: index.js (hunk 1 above)
Comment (bryce-seifert) [reply index.js]: fixed

File: index.js (hunk 2 above)
Comment (bryce-seifert) [reply index.js]: fixed

**Pull request merged**